Mage/src/main/java/mage/game/command/emblems/EmblemOfCard.java

```diff
@@ -0,0 +1,95 @@
+package mage.game.command.emblems;
+
+import mage.MageObject;
+import mage.abilities.AbilityImpl;
+import mage.cards.Card;
+import mage.cards.decks.DeckCardInfo;
+import mage.cards.mock.MockCard;
+import mage.cards.repository.CardCriteria;
+import mage.cards.repository.CardInfo;
+import mage.cards.repository.CardRepository;
+import mage.constants.Zone;
+import mage.game.command.Emblem;
+import mage.util.CardUtil;
+import org.apache.log4j.Logger;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class EmblemOfCard extends Emblem {
+    private boolean usesVariousArt;
+    private static final Logger logger = Logger.getLogger(EmblemOfCard.class);
+    public static Card lookupCard(
```

Comment:
private?

Reply:
This method was made when I noticed that `CardLookup.instance.lookupCardInfo` didn't really attempt to handle cards with multiple arts. I don't see any harm in letting other people use it if it fits their card-lookup needs.